### PR TITLE
Editorial updates to aria-controls content

### PR DESCRIPTION
**Pull request by scottaohara**
Making some revisions to how the `aria-controls` attribute was described here.  The reason for this is due to some misunderstandings that developers had, and referenced this as their source.  

Specifically, this modifies instances of content that people had been interpreting as requirements, toning down the language a bit to leave the intent, but to more align with the spec.

swaps out the text example of a link with aria-controls, which is valid – but generally one would want developers to use a button in this situation.  Rather, though also not particularly common, the example was replaced with a mention of `role=scrollbar` - besides combobox, the only other role in ARIA that actually _requires_ the use of aria-controls.

Finally, the tab widget markup example was modified to use classes instead of the `hidden` attribute.  For reasons I've already alluded to with other updates I've made to ARIA tab-related articles, it'd require far more explanation to indicate the situations which `hidden` would not be considered an anti-pattern for this widget.  Rather than explain that here, which would be out of scope, replacing with classes that invoke the same idea that the inactive tabpanels are completley hidden, seems far less work.

### CODE REVIEW

File: files/en-us/web/accessibility/aria/attributes/aria-controls/index.md [hunk 1]
```diff
@@ -14,15 +14,15 @@ The global `aria-controls` property identifies the element (or elements) whose c
...
 
 Other examples of controls include:
 
-- An accordion header button toggling the visibility of the accordion panel's content: the accordion header button would have `aria-controls` set to the ID of the element containing the accordion panel content.
-- A link with an icon that shows and hides a menu: the link would have `aria-controls` set to the ID of the element containing the menu.
-- A group of tabs that each display a different tabpanels: each element with [`role="tab"`](/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role) has the property `aria-controls` referring to its associated [`tabpanel`](/en-US/docs/Web/Accessibility/ARIA/Roles/tabpanel_role).
+- A the button parts of an accordion widget which toggle the visibility of their associated panel content. Each button may have an `aria-controls` specified, referencing the ID of the element containing the content associated with the invoking control.
```
Comment (estelle) [inline files/en-us/web/accessibility/aria/attributes/aria-controls/index.md:23]:
```suggestion
- The button parts of an accordion widget that toggle the visibility of their associated panel content. Each button may have an `aria-controls` specified, referencing the ID of the element containing the content associated with the invoking control.
```

File: files/en-us/web/accessibility/aria/attributes/aria-controls/index.md [hunk 2]
```diff
@@ -14,15 +14,15 @@ The global `aria-controls` property identifies the element (or elements) whose c
...
 
 Other examples of controls include:
 
-- An accordion header button toggling the visibility of the accordion panel's content: the accordion header button would have `aria-controls` set to the ID of the element containing the accordion panel content.
-- A link with an icon that shows and hides a menu: the link would have `aria-controls` set to the ID of the element containing the menu.
-- A group of tabs that each display a different tabpanels: each element with [`role="tab"`](/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role) has the property `aria-controls` referring to its associated [`tabpanel`](/en-US/docs/Web/Accessibility/ARIA/Roles/tabpanel_role).
+- The button parts of an accordion widget that toggle the visibility of their associated panel content. Each button may have an `aria-controls` specified, referencing the ID of the element containing the content associated with the invoking control.
+- An element with a role of [scrollbar](/en-US/docs/Web/Accessibility/ARIA/Roles/scrollbar_role): the scrollbar requires an `aria-controls` attribute referencing the ID of the element it controls.
```
Comment (estelle) [inline files/en-us/web/accessibility/aria/attributes/aria-controls/index.md:24]:
```suggestion
- An element with a role of [`scrollbar`](/en-US/docs/Web/Accessibility/ARIA/Roles/scrollbar_role): the scrollbar requires an `aria-controls` attribute referencing the ID of the element it controls.
```

File: files/en-us/web/accessibility/aria/attributes/aria-controls/index.md [hunk 3]
```diff
@@ -14,15 +14,15 @@ The global `aria-controls` property identifies the element (or elements) whose c
...
-- A link with an icon that shows and hides a menu: the link would have `aria-controls` set to the ID of the element containing the menu.
-- A group of tabs that each display a different tabpanels: each element with [`role="tab"`](/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role) has the property `aria-controls` referring to its associated [`tabpanel`](/en-US/docs/Web/Accessibility/ARIA/Roles/tabpanel_role).
+- The button parts of an accordion widget that toggle the visibility of their associated panel content. Each button may have an `aria-controls` specified, referencing the ID of the element containing the content associated with the invoking control.
+- An element with a role of [`scrollbar`](/en-US/docs/Web/Accessibility/ARIA/Roles/scrollbar_role): the scrollbar requires an `aria-controls` attribute referencing the ID of the element it controls.
+- A group of tabs that each display a different tabpanels: each element with [`role="tab"`](/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role) has an `aria-controls` attribute referencing its associated [`tabpanel`](/en-US/docs/Web/Accessibility/ARIA/Roles/tabpanel_role).
```
Comment (estelle) [inline files/en-us/web/accessibility/aria/attributes/aria-controls/index.md:25]:
```suggestion
- A group of tabs that each display a different tab panel: each element with [`role="tab"`](/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role) has an `aria-controls` attribute referencing its associated [`tabpanel`](/en-US/docs/Web/Accessibility/ARIA/Roles/tabpanel_role).
```

**Review by estelle — APPROVED** [review]
thank you!

**Comment by scottaohara** [comment]
thank you for the typo corrections :)

**Pull request merged**